### PR TITLE
Bump Citus 7.5.0

### PR DESCRIPTION
**Pull request by furkansahin**

### CODE REVIEW

File: META.json [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
    "description": "Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.",
-   "version": "7.4.1",
+   "version": "7.5.0",
    "maintainer": "\"Citus Data\" <packaging@citusdata.com>",
    "license": "agpl_3",
    "provides": {
       "citus": {
          "abstract": "Citus Distributed Database",
-         "file": "src/backend/distributed/citus--7.4-1.sql",
+         "file": "src/backend/distributed/citus--7.5-0.sql",
```
Comment (byucesoy) [inline META.json]: This should be 7.5-7, i.e. last sql file of that particular version.

**Review by byucesoy — CHANGES_REQUESTED** [review]
sql file needs to be fixed.

**Comment by furkansahin** [comment]
done.

**Review by byucesoy — APPROVED** [review]
:shipit:

**Pull request merged**